### PR TITLE
Add boot splash tutorial

### PR DESCRIPTION
**Pull request by ZX-WT**
Here is the tutorial with new images. Don't know whether they are clear or not.

### CODE REVIEW

File: tutorials/engine/boot_splash.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Go to scene>project settings>application>boot_splash
+
+.. image:: /img/boot_splash.png
+
+Uncheck the checkbox or choose a custom splash screen
+
+.. image:: /img/boot_splash_options.png
+
+Adding more splash screen
```
Comment (karroffel) [inline tutorials/engine/boot_splash.rst]: Shouldn't this be plural? "Adding more splash screens"

File: tutorials/engine/boot_splash.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+-----
+
+Godot is awesome, why not let everyone know about it?
+
+If you want to add one more splash screen after the engine's one, you can create a new scene with the splash screen and set it as the first scene and change to the next scene afterward.
+
+.. image:: /img/boot_splash_main_scene.png
+
+You can do this by creating a scene with a animationplayer node and use the finished() signal for changing the scene, remember to **enable autoplay** in the animationplayer.
```
Comment (karroffel) [inline tutorials/engine/boot_splash.rst]: with ***an*** AnimationPlayer node.

... the `finished()` signal ***to change the scene***. sounds better IMO.

Comment (karroffel) [inline tutorials/engine/boot_splash.rst]: Also the "the finished() signal" could use code formatting for the signal name.

\`finished()\`

File: tutorials/engine/boot_splash.rst [hunk 3]
```diff
@@ -11,7 +11,11 @@ Uncheck the checkbox or choose a custom splash screen
 
 .. image:: /img/boot_splash_options.png
 
+<<<<<<< HEAD
+Adding more splash screens
+=======
 Adding more splash screen
+>>>>>>> 18e8946ba7baa4bc5c71575d5f8ff8bb37361144
```
Comment (bojidar-bg) [inline tutorials/engine/boot_splash.rst]: Please, clean up merge conflict artefacts before PR-ing.

File: tutorials/engine/boot_splash.rst [hunk 4]
```diff
@@ -20,7 +24,11 @@ If you want to add one more splash screen after the engine's one, you can create
 
 .. image:: /img/boot_splash_main_scene.png
 
+<<<<<<< HEAD
+You can do this by creating a scene with an animationplayer node and use the 'finished()' signal to change the scene, remember to **enable autoplay** in the animationplayer.
+=======
 You can do this by creating a scene with a animationplayer node and use the finished() signal for changing the scene, remember to **enable autoplay** in the animationplayer.
+>>>>>>> 18e8946ba7baa4bc5c71575d5f8ff8bb37361144
```
Comment (bojidar-bg) [inline tutorials/engine/boot_splash.rst]: Same here.

**Comment by akien-mga** [comment]
Sorry about the delay, I hadn't merged that PR so far because I'm not fully convinced by the proposal (screenshots with blacked out project name and red dot for focus look unprofessional, and the actual tutorial text also lacks the "casual professionalism" we try to have in the docs as a whole), though I like the overall idea.

I will open an issue listing what I'd like to see in such a tutorial, referencing this PR for whoever would like to tackle the bigger picture.

**Comment by akien-mga** [comment]
I've described what I have in mind in #392.

**Pull request closed without merging**